Templates/SaaSFileTrigger-Batch/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "type": "apiHubFileTrigger",
+      "name": "input",
+      "direction": "in",
+      "path": "input-path-batch/{name}",
```

Comment:
I'd simplify this from "input-path-batch" to just "input-batch". Same for output below, and all other templates.